algopy_testing/src/algopy_testing/utils.py

```diff
@@ -76,5 +81,10 @@ def as_string(value: object) -> str:
             raise TypeError(f"value must be a string or String type, not {type(value).__name__!r}")
 
 
-def int_to_bytes(x: int) -> bytes:
-    return x.to_bytes((x.bit_length() + 7) // 8, "big")
+def int_to_bytes(x: int, pad_to: int | None = None) -> bytes:
+    result = x.to_bytes((x.bit_length() + 7) // 8, "big")
+    result = (
```

Comment:
should we prevent cases when pad_to is > than result ?

Reply:
I don't think we should. For example, `int_to_bytes(1, 8)` should return `b'\x00\x00\x00\x00\x00\x00\x00\x01'`.